lib/rex/post/meterpreter/packet.rb

```diff
@@ -314,6 +314,23 @@ def initialize(type, value = nil, compress=false)
     end
   end
 
+  def _tlv_type_string(value)
+    tlv_type = ::Rex::Post::Meterpreter::CommandMapper.find_cached_tlv_type_by_value(value)
+    ::Rex::Post::Meterpreter::CommandMapper.create_tlv_types_cache if tlv_type.nil?
+    tlv_type ||= ::Rex::Post::Meterpreter::CommandMapper.find_cached_tlv_type_by_value(value)
+
+    return if tlv_type.nil?
```

Comment:
Potentially we could just use the unknown default here?

```suggestion
    return "unknown-#{type}" if tlv_type.nil?
```